udata_front/theme/gouvfr/assets/js/pages/OrganizationPublishingForm/Step1CreateOrJoinOrganization.stories.ts

```diff
@@ -0,0 +1,57 @@
+import { withActions } from '@storybook/addon-actions/decorator';
+import type { Meta, StoryObj } from '@storybook/vue3';
+import { within, waitFor, userEvent } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+import Step1CreateOrJoinOrganization from './Step1CreateOrJoinOrganization.vue';
+import * as Stepper from '../../components/Form/Stepper/Stepper.stories';
```

Comment:
I think you can create a stepper array for the organization steps instead of using the dataset ones